web/src/views/Home.tsx

```diff
@@ -0,0 +1,30 @@
+//
+// Copyright © 2020 Province of British Columbia
+//
+// Licensed under the Apache License, Version 2.0 (the 'License');
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an 'AS IS' BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import styled from '@emotion/styled';
+import React from 'react';
+
+const Panel = styled.div`
```

Comment:
duplicate component with `NotFound.tsx`

Reply:
Fixed. The css part is drier now. I think the home page will become dashboard in the following features.